src/components/Widget/IntegrationsWidget.tsx

```diff
@@ -128,7 +160,102 @@ const IntegrationsWidget: FunctionComponent = () => {
                 ))}
               </Gallery>
             </CardBody>
-            {[COMMUNICATIONS, REPORTING, WEBHOOKS].includes(selectedTileValue) && (
+          </Card>
+        </>
+      ) : (
+        <Card ouiaId="integrations-widget" isFullHeight>
+          <CardBody className="pf-v5-u-pt-0">
+            <DataList aria-label="Integrations List" isCompact>
+              {integrationsData.map((integration, integrationIndex) => {
+                const isExpandable = integration.title !== 'Webhooks';
+                return (
+                  <React.Fragment key={`integration-fragment-${integrationIndex}`}>
+                    <DataListItem
+                      key={integrationIndex}
+                      aria-labelledby={`integration-${integrationIndex}`}
+                      isExpanded={isExpandable ? expandedIndex.includes(integrationIndex) : false}
+                    >
+                      <DataListItemRow>
+                        {isExpandable ? (
+                          <DataListToggle
+                            onClick={() => onToggle(integrationIndex, isExpandable)}
+                            isExpanded={expandedIndex.includes(integrationIndex)}
+                            id={`toggle-${integrationIndex}`}
+                            aria-controls={`expand-${integrationIndex}`}
+                          />
+                        ) : (
+                          <div style={{ width: '47px' }} />
```

Comment:
Any chance to make this magic px number be less magic? Any utility class we can leverage?

Reply:
@epwinchell do you have any suggestions?